src/dotnet/MsbuildProject.cs

```diff
@@ -1,17 +1,17 @@
 ﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
```

Comment:
I prefer alphabetic one, but I don't want to hold this PR on this. I am going to merge this.